reproducible/ubuntu/node_gen_tar.bzl

```diff
@@ -73,17 +73,23 @@ load(
     "docker_build",
 )
 
-def nodebootstrap_image(name, env=None):
+def node_gen_tar_rule(name, env=None):
```

Comment:
nit: make this node_gen_tar and make the other one _node_gen_tar

Reply:
done